source/docs/advanced/features/display-a-map.md

```diff
@@ -5,11 +5,38 @@ title: Display a map
 
 Front-Commerce has a generic map component that can be used by modules to display maps. Since we are well aware that different shops come with different needs, we provide several implementations out of the box:
 
-- [Open Street Map with Leaflet](#Open-Street-Map-OSM-with-Leaflet)
-- [Google Maps](#Google-Maps)
 
-The goal is to choose one to make sure that all maps accross the website look the same.
 
+- [Open Street Map (OSM) with Leaflet](#open-street-map-osm-with-leaflet)
+- [Google Maps](#google-maps)
+
+The goal is to choose one to make sure that all maps across the website look the same.
+
+
+### Component Props  <!-- omit in toc -->
+
+We have harmonized the props of the map components to make sure that they are consistent across different components.
+#### Map Component
+
+| Property        | Description                              | Type                   |
+|-----------------|------------------------------------------|------------------------|
+| locations       | A list of locations used for the markers | [`LocationInputShape[]`](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/components/organisms/Map/location.js#L22-31) |
+| getMarker       | The marker node for locations            | `(location) => ReactNode` |
+| zoom            | Default zoom level                       | `number`               |
+| defaultBounds   | The default map bounds                   | [`CoordinatesShape[]`](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/components/organisms/Map/location.js#L17-20)   |
+| defaultCenter   | The default center for the map.          | [`CoordinatesShape` ](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/components/organisms/Map/location.js#L17-20)    |
+| onBoundsChanged | Event handler for bound changes          | `(event) => void`      |
+
+
+#### Marker Component
+
+| Property      | Description                            | Type                 |
+|---------------|----------------------------------------|----------------------|
+| location      | Location for the marker                | [`LocationInputShape` ](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/components/organisms/Map/location.js#L22-31)|
```

Comment:
**REMINDER:** update the link lines when [MR914](https://gitlab.com/front-commerce/front-commerce/-/merge_requests/914) has been merged